File: beginner/java/1930.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);
        
        int n1, n2, n3, n4;
    
		String[] valores;
		valores = entrada.nextLine().split(" ");

		n1 = Integer.parseInt(valores[0]);
		n2 = Integer.parseInt(valores[1]);
		n3 = Integer.parseInt(valores[2]);
		n4 = Integer.parseInt(valores[3]);
        
        System.out.printf("%d%n", ((n1 + n2 + n3 + n4) - 3));
    }
}

File: beginner/javascript/1930.js
var input = require('fs').readFileSync('stdin', 'utf8');

var linhas = input.split('\n');

var valores = linhas[0].split(' ');

console.log((parseInt(valores[0]) + parseInt(valores[1]) + parseInt(valores[2]) + parseInt(valores[3])) - 3)